st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -305,6 +305,12 @@
                 'type': 'boolean',
                 'default': False
             },
+            'verify_ssl_cert': {
+                'description': 'SSL verification is not supported.',
+                'type': 'boolean',
+                'default': False,
+                'immutable': True
```

Comment:
I like default = False and allow it to be mutable. 

You could also add cacert but it has to be content and not path. I'd think this is not a frequent use case. We can add it if we later need it.